for stacked areas, only add trace if not all are zero

markushal opened the pull request:
fixes #416

joAschauer reviewed (approved):
looks good. Although I get deprecation warnings for `go.Line`:

```
plotly.graph_objs.Line is deprecated.
Please replace it with one of the following more specific types
  - plotly.graph_objs.scatter.Line
  - plotly.graph_objs.layout.shape.Line
  - etc.
```

markushal commented:
> looks good. Although I get deprecation warnings for `go.Line`:
> 
> ```
> plotly.graph_objs.Line is deprecated.
> Please replace it with one of the following more specific types
>   - plotly.graph_objs.scatter.Line
>   - plotly.graph_objs.layout.shape.Line
>   - etc.
> ```

Thanks for pointing this out. I changed the code to remove the warning, and I also updated plotly to the latest version in ``requirements.txt``.
